src/common/filter/Expressions.h

```diff
@@ -838,7 +840,7 @@ class TypeCastingExpression final : public Expression {
 class ArithmeticExpression final : public Expression {
 public:
     enum Operator : uint8_t {
```

Comment:
enum class was more better.

Reply:
Don't want to modify the origin code in this PR if not necessary.